recipes/genome-properties/build.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+## perl version
+perl_version=$(perl -e 'print $^V');
+perl_version=${perl_version:1}
+
+PERLLIB="${PREFIX}/lib/perl5/${perl_version}/perl_lib"
+mkdir -p $PERLLIB
+
+mkdir -p "$PREFIX/bin"
+mkdir -p "$PREFIX/code"
+mkdir -p "$PREFIX/data"
+mkdir -p "$PREFIX/flatfiles"
+
+cp -r ./code/scripts/* "$PREFIX/bin"
+cp -r ./code/* "$PREFIX/code"
+cp -r ./data/* "$PREFIX/data"
+cp -r ./flatfiles/* "$PREFIX/flatfiles"
+
+if [[ ${HOST} =~ .*darwin.* && ("$HOME" == 'UNKNOWN' || -z "$HOME") ]]; then
+    export HOME=/tmp
+fi
+
+cpanm -l $PERLLIB --force DBIx::Class
+cpanm -l $PERLLIB --force Data::Printer
```

Comment:
Please don't do that. We need those packages in conda and depend on them.

Reply:
Sorry, I used examples like these for inspiration (see [here](https://github.com/bioconda/bioconda-recipes/blob/14ad6069028b000c339cc10c4974294a1fda4a5d/recipes/perl-atlas-modules/build.sh) and [here](https://github.com/bioconda/bioconda-recipes/blob/84623f36913308617c627b79759877398eda2730/recipes/perl-bio-phylo-forest-dbtree/build.sh#L4C1-L4C28))... I understand it's not the best approach but unfortunately these packages aren't on bioconda yet, if we have to be push these to bioconda first I'm not sure we have the resources for that at the moment 🫤 Let me know if there's a better way at the moment though!

Reply:
Since those packages are not Bio related they should be added to `conda-forge` instead.
See any PR with passing CI at https://github.com/conda-forge/staged-recipes/pulls for inspiration.

Reply:
I understand, but if we don't get people that care about the perl ecosystem we can not support this in the long run.